app/models/authorized_user.rb

```diff
@@ -0,0 +1,26 @@
+# == Schema Information
+#
+# Table name: Authorized_users
+#
+#  id                     :integer          not null, primary key
+#  email                  :string           default(""), not null
+#  role                   :string           default("coach"), not null
+#
+
+class Authorized_user < ActiveRecord::Base
+
+  ADMIN = "admin"
+  COACH = "coach"
+
+  def self.has_email?(email)
+    if self.find_by(email: email).nil?
```

Comment:
`find_by` is not a method of a particular model object, it's a class method
so this function can be replaced with a one liner `!AuthorizedUser.find_by_email(email).nil?`